docker/init.sh

```diff
@@ -8,6 +8,9 @@ docker-compose run __PLUGIN_LOWERCASE__ ant -f __SHOPWARE_ROOT__build/build.xml
 
 # Install plugin
 docker-compose run __PLUGIN_LOWERCASE__ php __SHOPWARE_ROOT__bin/console sw:plugin:refresh --env="__ENV__"
+
+I: ./vendor/shopware/plugin-dev-tools/bin/exec-before-install.sh __PLUGIN_LOWERCASE__ __ENV__
```

Comment:
I am not sure if this could work, I will try it on my machine, but maybe it fails because is not running inside the script inside the docker container